python/core/auto_generated/qgsvectorfilewritertask.sip.in

```diff
@@ -41,7 +41,7 @@ and save ``options``.
 
   signals:
 
-    void writeComplete( const QString &newFilename );
+    void writeComplete( const QString &newFilename, const QString &newLayer );
```

Comment:
This is also a break - you'll need to add a second signal and deprecate this one.

Reply:
@nyalldawson , will I need to emit both signals in QgsVectorFileWriterTask::finished() ?